### ISSUE TITLE
ZAP failing to launch on Kali

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zaproxy#5526]
Not confirmed yet, but reported twice now:

- [User group](https://groups.google.com/d/msg/zaproxy-users/xTR1NYAc8s8/m8qNyzucAgAJ)
- [Reddit](https://www.reddit.com/r/AskNetsec/comments/cqph27/owasp_zap_not_starting/)

Raised this issue in order to track whatever we find out.

**Comment by sh1va73** [comment]
After launchin ZAP it hangs again. I did a ps but found no zap in the list. I did kill -3 java to stop ZAP:

"ZAP-CallbackServer" #33 daemon prio=6 os_prio=0 tid=0x0000560b2d646000 nid=0x5d4 runnable [0x00007f4f30b55000]
   java.lang.Thread.State: RUNNABLE
	at java.net.PlainSocketImpl.socketAccept(Native Method)
	at java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:409)
	at java.net.ServerSocket.implAccept(ServerSocket.java:545)
	at java.net.ServerSocket.accept(ServerSocket.java:513)
	at org.parosproxy.paros.core.proxy.ProxyServer.run(ProxyServer.java:268)
	at java.lang.Thread.run(Thread.java:748)

"MultiThreadedHttpConnectionManager cleanup" #32 daemon prio=6 os_prio=0 tid=0x00007f4f58109800 nid=0x5d2 in Object.wait() [0x00007f4f30e81000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
	- locked <0x00000000db611880> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
	at org.apache.commons.httpclient.MultiThreadedHttpConnectionManager$ReferenceQueueThread.run(MultiThreadedHttpConnectionManager.java:1122)

"ZAP-HUD-Tutorial" #30 daemon prio=6 os_prio=0 tid=0x0000560b2d61b800 nid=0x5d0 runnable [0x00007f4f31283000]
   java.lang.Thread.State: RUNNABLE
	at java.net.PlainSocketImpl.socketAccept(Native Method)
	at java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:409)
	at java.net.ServerSocket.implAccept(ServerSocket.java:545)
	at java.net.ServerSocket.accept(ServerSocket.java:513)
	at org.parosproxy.paros.core.proxy.ProxyServer.run(ProxyServer.java:268)
	at java.lang.Thread.run(Thread.java:748)

"ZAP-ScriptChangeOnDiskThread" #29 daemon prio=6 os_prio=0 tid=0x0000560b2d40d800 nid=0x5ce waiting on condition [0x00007f4f31784000]
   java.lang.Thread.State: TIMED_WAITING (sleeping)
	at java.lang.Thread.sleep(Native Method)
	at org.zaproxy.zap.extension.scripts.ConsolePanel$1.run(ConsolePanel.java:236)

"ZAP-PassiveScanner" #28 daemon prio=6 os_prio=0 tid=0x0000560b2d3a0800 nid=0x5cd waiting on condition [0x00007f4f31a85000]
   java.lang.Thread.State: TIMED_WAITING (sleeping)
	at java.lang.Thread.sleep(Native Method)
	at org.zaproxy.zap.extension.pscan.PassiveScanThread.run(PassiveScanThread.java:153)

"ZAP-DownloadManager" #26 daemon prio=6 os_prio=0 tid=0x0000560b2cb7f800 nid=0x5ca waiting on condition [0x00007f4f3582e000]
   java.lang.Thread.State: TIMED_WAITING (sleeping)
	at java.lang.Thread.sleep(Native Method)
	at org.zaproxy.zap.extension.autoupdate.DownloadManager.run(DownloadManager.java:118)

"Thread-6" #25 daemon prio=5 os_prio=0 tid=0x00007f4f44ec0800 nid=0x5c4 runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"ZAP-BootstrapGUI" #24 prio=6 os_prio=0 tid=0x00007f4f3c3c4800 nid=0x5c9 waiting on condition [0x00007f4f3592c000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x00000000d9a6e518> (a java.util.concurrent.FutureTask)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.FutureTask.awaitDone(FutureTask.java:429)
	at java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at org.GNOME.Accessibility.AtkUtil.invokeInSwing(AtkUtil.java:68)
	at org.GNOME.Accessibility.AtkObject.hashCode(AtkObject.java:234)
	at org.GNOME.Accessibility.AtkWrapper.emitSignal(Native Method)
	at org.GNOME.Accessibility.AtkWrapper$5.propertyChange(AtkWrapper.java:545)
	at java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:335)
	at java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:327)
	at java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:263)
	at javax.accessibility.AccessibleContext.firePropertyChange(AccessibleContext.java:768)
	at javax.swing.JLabel.setText(JLabel.java:338)
	at javax.swing.plaf.synth.SynthComboBoxUI$SynthComboBoxRenderer.getListCellRendererComponent(SynthComboBoxUI.java:493)
	at org.zaproxy.zap.extension.httppanel.component.HttpPanelComponentViewsManager$CustomDelegateListCellRenderer.getListCellRendererComponent(HttpPanelComponentViewsManager.java:596)
	at org.zaproxy.zap.extension.httppanel.component.HttpPanelComponentViewsManager$CustomDelegateListCellRenderer.getListCellRendererComponent(HttpPanelComponentViewsManager.java:556)
	at javax.swing.plaf.basic.BasicComboBoxUI.getDisplaySize(BasicComboBoxUI.java:1353)
	at javax.swing.plaf.basic.BasicComboBoxUI.getMinimumSize(BasicComboBoxUI.java:921)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1744)
	at java.awt.FlowLayout.minimumLayoutSize(FlowLayout.java:463)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.BorderLayout.minimumLayoutSize(BorderLayout.java:663)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.FlowLayout.minimumLayoutSize(FlowLayout.java:463)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.BorderLayout.minimumLayoutSize(BorderLayout.java:658)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.BorderLayout.minimumLayoutSize(BorderLayout.java:668)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.BorderLayout.minimumLayoutSize(BorderLayout.java:668)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at javax.swing.plaf.basic.BasicTabbedPaneUI$TabbedPaneLayout.calculateSize(BasicTabbedPaneUI.java:2303)
	at javax.swing.plaf.basic.BasicTabbedPaneUI$TabbedPaneLayout.minimumLayoutSize(BasicTabbedPaneUI.java:2282)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.BorderLayout.minimumLayoutSize(BorderLayout.java:663)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at javax.swing.plaf.basic.BasicSplitPaneUI$BasicHorizontalLayoutManager.minimumLayoutSize(BasicSplitPaneUI.java:1451)
	at javax.swing.plaf.basic.BasicSplitPaneUI.getMinimumSize(BasicSplitPaneUI.java:1091)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1744)
	at javax.swing.plaf.basic.BasicSplitPaneUI$BasicHorizontalLayoutManager.minimumLayoutSize(BasicSplitPaneUI.java:1451)
	at javax.swing.plaf.basic.BasicSplitPaneUI.getMinimumSize(BasicSplitPaneUI.java:1091)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1744)
	at java.awt.CardLayout.minimumLayoutSize(CardLayout.java:311)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at java.awt.CardLayout.minimumLayoutSize(CardLayout.java:311)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.minimumSize(Container.java:1851)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1746)
	at javax.swing.BoxLayout.checkRequests(BoxLayout.java:483)
	at javax.swing.BoxLayout.layoutContainer(BoxLayout.java:425)
	- locked <0x00000000d5413738> (a javax.swing.BoxLayout)
	at java.awt.Container.layout(Container.java:1513)
	at java.awt.Container.doLayout(Container.java:1502)
	at java.awt.Container.validateTree(Container.java:1698)
	at java.awt.Container.validateTree(Container.java:1707)
	at java.awt.Container.validateTree(Container.java:1707)
	at java.awt.Container.validateTree(Container.java:1707)
	at java.awt.Container.validate(Container.java:1633)
	- locked <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at org.parosproxy.paros.extension.ExtensionLoader.hookAllExtension(ExtensionLoader.java:924)
	at org.parosproxy.paros.extension.ExtensionLoader.startLifeCycle(ExtensionLoader.java:782)
	at org.parosproxy.paros.control.AbstractControl.loadExtension(AbstractControl.java:58)
	at org.parosproxy.paros.control.Control.init(Control.java:132)
	at org.parosproxy.paros.control.Control.initSingletonWithView(Control.java:345)
	at org.zaproxy.zap.GuiBootstrap.initControlAndPostViewInit(GuiBootstrap.java:270)
	at org.zaproxy.zap.GuiBootstrap.access$100(GuiBootstrap.java:68)
	at org.zaproxy.zap.GuiBootstrap$2.run(GuiBootstrap.java:208)
	at java.lang.Thread.run(Thread.java:748)

"TimerQueue" #20 daemon prio=5 os_prio=0 tid=0x00007f4f3c370000 nid=0x5c7 waiting on condition [0x00007f4f34a52000]
   java.lang.Thread.State: WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x00000000d5131000> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
	at java.util.concurrent.DelayQueue.take(DelayQueue.java:211)
	at javax.swing.TimerQueue.run(TimerQueue.java:174)
	at java.lang.Thread.run(Thread.java:748)

"Timer-0" #18 daemon prio=6 os_prio=0 tid=0x00007f4f3c320800 nid=0x5c6 in Object.wait() [0x00007f4f34d53000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.util.TimerThread.mainLoop(Timer.java:552)
	- locked <0x00000000d5131200> (a java.util.TaskQueue)
	at java.util.TimerThread.run(Timer.java:505)

"HSQLDB Timer @bd21f5" #16 daemon prio=6 os_prio=0 tid=0x00007f4f3c199800 nid=0x5c3 in Object.wait() [0x00007f4f36c4e000]
   java.lang.Thread.State: TIMED_WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at org.hsqldb.lib.HsqlTimer$TaskQueue.park(Unknown Source)
	- locked <0x00000000d2db0548> (a org.hsqldb.lib.HsqlTimer$TaskQueue)
	at org.hsqldb.lib.HsqlTimer.nextTask(Unknown Source)
	- locked <0x00000000d2db0548> (a org.hsqldb.lib.HsqlTimer$TaskQueue)
	at org.hsqldb.lib.HsqlTimer$TaskRunner.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:748)

"DestroyJavaVM" #15 prio=5 os_prio=0 tid=0x00007f4f6c00b000 nid=0x5ae waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"AWT-EventQueue-1" #13 prio=6 os_prio=0 tid=0x00007f4f6c619000 nid=0x5c2 waiting for monitor entry [0x00007f4f3734d000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at java.awt.Container.dispatchEventImpl(Container.java:2299)
	- waiting to lock <0x00000000d2456e48> (a java.awt.Component$AWTTreeLock)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:760)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:733)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:730)
	at org.GNOME.Accessibility.AtkWrapper$6.dispatchEvent(AtkWrapper.java:705)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

"AWT-Shutdown" #14 prio=5 os_prio=0 tid=0x00007f4f6c618000 nid=0x5c1 in Object.wait() [0x00007f4f37450000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:502)
	at sun.awt.AWTAutoShutdown.run(AWTAutoShutdown.java:295)
	- locked <0x00000000d2676178> (a java.lang.Object)
	at java.lang.Thread.run(Thread.java:748)

"process reaper" #12 daemon prio=10 os_prio=0 tid=0x00007f4f6c5e4000 nid=0x5c0 waiting on condition [0x00007f4f37bbf000]
   java.lang.Thread.State: TIMED_WAITING (parking)
	at sun.misc.Unsafe.park(Native Method)
	- parking to wait for  <0x00000000d2676338> (a java.util.concurrent.SynchronousQueue$TransferStack)
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

"AWT-XAWT" #11 daemon prio=6 os_prio=0 tid=0x00007f4f6c5cf000 nid=0x5be runnable [0x00007f4f37cc0000]
   java.lang.Thread.State: RUNNABLE
	at sun.awt.X11.XToolkit.waitForEvents(Native Method)
	at sun.awt.X11.XToolkit.run(XToolkit.java:574)
	at sun.awt.X11.XToolkit.run(XToolkit.java:538)
	at java.lang.Thread.run(Thread.java:748)

"Java2D Disposer" #9 daemon prio=10 os_prio=0 tid=0x00007f4f6c5ae000 nid=0x5bd in Object.wait() [0x00007f4f37fd4000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
	- locked <0x00000000d2676730> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
	at sun.java2d.Disposer.run(Disposer.java:148)
	at java.lang.Thread.run(Thread.java:748)

"Service Thread" #8 daemon prio=9 os_prio=0 tid=0x00007f4f6c180800 nid=0x5bb runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C1 CompilerThread1" #7 daemon prio=9 os_prio=0 tid=0x00007f4f6c17d800 nid=0x5ba waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" #6 daemon prio=9 os_prio=0 tid=0x00007f4f6c17b000 nid=0x5b9 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" #5 daemon prio=9 os_prio=0 tid=0x00007f4f6c179000 nid=0x5b8 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Surrogate Locker Thread (Concurrent GC)" #4 daemon prio=9 os_prio=0 tid=0x00007f4f6c178000 nid=0x5b7 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" #3 daemon prio=8 os_prio=0 tid=0x00007f4f6c141800 nid=0x5b6 in Object.wait() [0x00007f4f5c1b6000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
	- locked <0x00000000d26770a8> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:216)

"Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x00007f4f6c13f000 nid=0x5b5 in Object.wait() [0x00007f4f5c2b7000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
	- locked <0x00000000d2677260> (a java.lang.ref.Reference$Lock)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)

"VM Thread" os_prio=0 tid=0x00007f4f6c135000 nid=0x5b4 runnable 

"Gang worker#0 (Parallel GC Threads)" os_prio=0 tid=0x00007f4f6c01f800 nid=0x5af runnable 

"G1 Main Concurrent Mark GC Thread" os_prio=0 tid=0x00007f4f6c05f000 nid=0x5b2 runnable 

"Gang worker#0 (G1 Parallel Marking Threads)" os_prio=0 tid=0x00007f4f6c061000 nid=0x5b3 runnable 

"G1 Concurrent Refinement Thread#0" os_prio=0 tid=0x00007f4f6c023800 nid=0x5b1 runnable 

"G1 Concurrent Refinement Thread#1" os_prio=0 tid=0x00007f4f6c022000 nid=0x5b0 runnable 

"VM Periodic Task Thread" os_prio=0 tid=0x00007f4f6c183800 nid=0x5bc waiting on condition 

JNI global references: 3977

Heap
 garbage-first heap   total 221184K, used 150783K [0x00000000d2200000, 0x00000000d23006c0, 0x0000000100000000)
  region size 1024K, 103 young (105472K), 6 survivors (6144K)
 Metaspace       used 57989K, capacity 60070K, committed 60436K, reserved 1099776K
  class space    used 8596K, capacity 9203K, committed 9264K, reserved 1048576K

**Comment by psiinon** [comment]
@sh1va73 many thanks!

**Comment by sh1va73** [comment]
I tried again with ps aux | less at same time I boot ZAP. Now I see a running process which I can do kill -3 PID. But where do I find the output of that?

**Comment by thc202** [comment]
The thread dump provided earlier is enough (the output would be in the console that started ZAP, if any).

**Comment by kingthorin** [comment]
Tested on a Kali VM:
`root@kali:~# java -version openjdk version "11.0.4" 2019-07-16 OpenJDK Runtime Environment (build 11.0.4+11-post-Debian-1) OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Debian-1, mixed mode)`
Help > Support Info: `Java Version: Debian 11.0.4`

Checked `/etc/java-11-openjdk/accessibility.properties` which contains the line `#assistive_technologies=org.GNOME.Accessibility.AtkWrapper`. 

When I un-comment the line then zap fails to start, when I re-comment it then things are fine.

**Comment by sh1va73** [comment]
> Tested on a Kali VM:
> `root@kali:~# java -version openjdk version "11.0.4" 2019-07-16 OpenJDK Runtime Environment (build 11.0.4+11-post-Debian-1) OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Debian-1, mixed mode)`
> Help > Support Info: `Java Version: Debian 11.0.4`
> 
> Checked `/etc/java-11-openjdk/accessibility.properties` which contains the line `#assistive_technologies=org.GNOME.Accessibility.AtkWrapper`.
> 
> When I un-comment the line then zap fails to start, when I re-comment it then things are fine.

I tested this solution on Java version 8, and this solution works as well.
Now ZAP starts again. Thanks for this solution!

**Comment by psiinon** [comment]
Note that I've raised this in the kali tracker: https://bugs.kali.org/view.php?id=5659

**Comment by sh1va73** [comment]
Thanks for the follow up!

**Comment by thc202** [comment]
#5545 addresses more issues seen in Kali.

**Comment by blshkv** [comment]
Please do a proper release (we relay ```ZAP Core``` tar ball) for all platform. Kali uses a common openjdk which is available on any other distros so other platforms may be (Pentoo is not) affected too.

**Comment by kingthorin** [comment]
@blshkv It's unclear if you mean the core zip or Linux tar ball. If the latter: https://github.com/zaproxy/zaproxy/releases/tag/v2.8.1

**Comment by blshkv** [comment]
The previous release (https://github.com/zaproxy/zaproxy/releases/tag/v2.8.0) had more options
including ```ZAP_2.8.0_Core.zip``` minimalistic archive. We use that as a base and provide plugins as and additional option for our users.

**Comment by psiinon** [comment]
@blshkv does ZAP 2.8.0 start for you? If so then you dont need this patch.
If it does not then please raise a new issue.
The core release is just the linux tar with some add-ons removed - if you want to take the linux tar and remove some add-ons then that will have the same effect. Or you can just patch the 2.8.1 ZAP core jar into the previous core tar.
A full release on all platforms is a significant amount of work for us, and unfortunately one that we cannot resource at this time.

**Comment by thc202** [comment]
@blshkv the problem is not triggered by OpenJDK but [Java ATK Wrapper](https://wiki.gnome.org/Accessibility/JavaAtkWrapper).
